packages/@webex/plugin-meetings/src/rtcMetrics/index.ts

```diff
@@ -95,7 +96,7 @@ export default class RtcMetrics {
         // this is the first useful set of data (WCME gives it to us after 5s), send it out immediately
         // in case the user is unhappy and closes the browser early
         this.sendMetricsInQueue();
-        this.shouldSendMetricsOnNextStatsReport = false;
+        this.shouldSendMetricsOnNextStatsReport -= 1;
```

Comment:
maybe worth changing the condition in line 95 to `this.shouldSendMetricsOnNextStatsReport > 0` just in case if that variable ever became negative somehow?